table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -183,11 +185,13 @@ public void sync(JournalContext context) throws IOException {
         Journal.JournalEntry entry = Journal.JournalEntry.newBuilder().setAddTable(addTableEntry)
             .build();
         applyAndJournal(context, entry);
+        returnVal = true;
       } else {
         // sync metadata from udb
         // TODO(gpang): implement
```

Comment:
we will need to discover new partitions in this part.

Reply:
yeah, it is.. it goes through the Udb and if the table is not found in the correct table collection, it adds the table.

Reply:
oops, you mean new partitions..